test/unit/TokenController/setup.js

```diff
@@ -10,18 +10,20 @@ async function setup() {
   const { internalContracts, members } = accounts;
   const internal = internalContracts[0];
 
-  const pooledStaking = await ethers.deployContract('TCMockPooledStaking');
-
   const stakingPoolFactory = await ethers.deployContract('StakingPoolFactory', [accounts.defaultSender.address]);
+  const stakingNFT = await ethers.deployContract('TCMockStakingNFT');
 
   const nxm = await ethers.deployContract('NXMTokenMock');
-  const tokenController = await ethers.deployContract('TokenController', [
+  const tokenController = await ethers.deployContract('DisposableTokenController', [
```

Comment:
I'm not sure if this is the correct use of `DisposableTokenController`. 

`DisposableTokenController` inherits from `TokenController` but adds extra functionality that let's me simulate locking of v1 claim assessment tokens and v1 cover notes.